src/main/java/nl/novi/techiteasycontroller/Controller/ExceptionController.java

```diff
@@ -0,0 +1,24 @@
+package nl.novi.techiteasycontroller.Controller;
+
+
+import nl.novi.techiteasycontroller.Exceptions.IndexOutOfBounds;
```

Comment:
Hier had je de standaard IndexOutOfBoundsException mogen gebruiken. Deze zit in het standaard java.lang package en hoeft niet geimporteerd te worden